recipe/meta.yaml

```diff
@@ -1,34 +1,37 @@
 {% set name = "metoffice-afterburner" %}
-{% set version = "1.3.3" %}
-{% set sha256 = "c44f0b3dca9e4833e4064aae3e84a9da43b68483fe942e7ae2c6bfbe0c3a5670" %}
+{% set version = "1.3.4" %}
+{% set sha256 = "ed2bf255316587d4cc7dd418a0e035fbe7b142d3d7861f610f6497b94b08a002" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name | replace('-', '_') }}-{{ version }}.tar.gz
   sha256: {{ sha256 }}
 
 build:
   number: 0
   script: "{{ PYTHON }} -m pip install . -vv"
-  skip: True  # [win or osx]
+  skip: true  # [win or osx]
 
 requirements:
   host:
     - rsync
     - python
     - pip
+    - setuptools
   run:
     - python
     - iris >=2.4
     - pyparsing >=2
     - windspharm
+    - setuptools
 
 test:
   imports:
     - afterburner
+    - setuptools
```

Comment:
Does it work without this `setuptools`?

Reply:
[18cf505](https://github.com/conda-forge/metoffice-afterburner-feedstock/pull/14/commits/18cf505349653f2245e1a3d9623a4c165492d304), Yup, things have passed!